packaging/hipify-clang.rpm_postun

```diff
@@ -11,4 +10,6 @@ if [ $1 -eq 0 ]; then
   rm -f @ROCMBINDIR@/findcode.sh
   rm -f @ROCMBINDIR@/finduncodep.sh
   rmdir --ignore-fail-on-non-empty @ROCMBINDIR@
+  rmdir --ignore-fail-on-non-empty @HIPBINDIR@
+  rmdir --ignore-fail-on-non-empty ${HIPDIR}
```

Comment:
You may use `@CPACK_PACKAGING_INSTALL_PREFIX@` here instead of `HIPDIR` as it is obviously the HIP/HIPIFY root installation dir.